HaruhiHeiretsuEditor.sln

```diff
@@ -0,0 +1,43 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
why is the sln changed? that's weird right? is it bc you opened it with VS 2019? in any case, please revert this file before merging

Reply:
Oh, whoops.